apps/mobile/src/features/balances/token-balance.tsx

```diff
@@ -1,16 +1,29 @@
 import { ReactNode } from 'react';
 
 import { Balance } from '@/components/balance/balance';
+import { t } from '@lingui/macro';
 
-import { Money } from '@leather.io/models';
+import { CryptoAssetProtocol, Money } from '@leather.io/models';
 import { Flag, ItemLayout, Pressable } from '@leather.io/ui/native';
 
+export function getChainLayerFromAssetProtocol(protocol: CryptoAssetProtocol) {
```

Comment:
Nice work! I guess this changed again in designs? 

Before we had to say nothing for `BTC` / `STX` then always say `X blockchain` for others